.storybook/storybook.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Barlow');
```

Comment:
I've added this so I could preview the Greeter using the right font in Storybook. The font is obviously not included in the hsds-react package.

Reply:
Perfect!